src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumesRepository.cs

```diff
@@ -78,23 +78,51 @@ public class ResumesRepository : IResumesRepository
                     CreationDate = DateTime.Now.AddDays(-10),
                     LastUpdate = DateTime.Now
                 }
-            };
+            }.ToList();
 
-        public Resume GetResume(Guid id)
+        public Resume Add(Resume newObject)
         {
-            Resume result = resumes.FirstOrDefault(p => Equals(p.Id, id));
-            return result;
+            Resumes.Add(newObject);
+            return Resumes.Last();
+        }
+
+        public void Delete(Guid id)
+        {
+            throw new NotImplementedException();
+        }
+
+        public List<Resume> GetAll()
+        {
+            return Resumes.ToList();
```

Comment:
Resumes is a list already

Reply:
Yes, in case there is a need to enter more information into our GetAll method.

Reply:
I think return Resumes is enough